CMakeLists.txt

```diff
@@ -717,7 +717,7 @@ add_custom_target(lrelease
 ##  Create and install the plugin framework manual
 ##
 
-add_custom_command(OUTPUT ${PROJECT_BINARY_DIR}/plugins
-   COMMAND genManual ${PROJECT_SOURCE_DIR} ${PROJECT_BINARY_DIR}
-   DEPENDS genManual
-   )
+#add_custom_command(OUTPUT ${PROJECT_BINARY_DIR}/plugins
+#   COMMAND genManual ${PROJECT_SOURCE_DIR} ${PROJECT_BINARY_DIR}
+#   DEPENDS genManual
+#   )
```

Comment:
Why? Where does that manual get generated now?

Reply:
Not sure where this came from.  I didn't add it.  Might have been generated by a script or by QtCreator.  I'm not sure how to tell git to ignore my local changes to this file.

Reply:
Got to have been you changing it.
Prevent it by simply by not commint it.